src/server/req_quejob.c

```diff
@@ -310,10 +309,10 @@ req_quejob(struct batch_request *preq)
 	resource_def	*prdefplc;
 	resource	*presc;
 	conn_t		*conn;
-	attribute       temp_attr;
 #else
 	mom_hook_input_t  hook_input;
 	mom_hook_output_t hook_output;
+	char	namebuf[MAXPATHLEN+1];
```

Comment:
indentation is off

Reply:
Fixed in new commit rc2 https://github.com/PBSPro/pbspro/pull/892/commits/90b8b4df18442ed3f8016032e9c1cc43f83cdf5f